### PR TITLE
Console Log UI for garfana/loki 

### PR DESCRIPTION
**Pull request by hisarbalik**
Simple grafana/loki log view for kyma console, log ui provide minimal functionality of searching and filtering logs for kyma/kubernetes component for kyma console.  

Open topics:

- UX design
- Modularising of UI, ui depend on optional grafana/loki deployment

### CODE REVIEW

File: logs/src/assets/config/config.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var domain = 'kyma.local';
```
Comment (a-thaler) [inline logs/src/assets/config/config.js]: Please double-check if it should be on gitignore

File: README.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ The Console project consists of the following UI projects:
 - [`Brokers`](./brokers) - The view for displaying Service Brokers
 - [`Lambda`](./lambda) - The view for lambda functions
 - [`Content`](./content) - The documentation view
+- [`Log UI`](./content) - The logs view
```
Comment (a-thaler) [inline README.md]: Maybe name the folder itself "log-ui" as well?

**Comment by a-thaler** [comment]
Test-infra prow job added in PR https://github.com/kyma-project/test-infra/pull/587

File: log-ui/src/assets/config/config.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+var domain = 'kyma.local';
```
Comment (magicmatatjahu) [inline log-ui/src/assets/config/config.js]: Please remove content from this file and push only empty file. This file is fulled by lerna `start` script.

File: log-ui/src/app/search-form/search-form.component.html [hunk 4]
```diff
@@ -1,5 +1,12 @@
 <div class="fd-section">
-<div class="fd-panel">
+  <div class="fd-panel">
+  <div class="fd-panel__header">
```
Comment (jesusreal) [inline log-ui/src/app/search-form/search-form.component.html]: please fix html indentation here

**Comment by jesusreal** [comment]
Regarding UI tests, as part of integrating Fiori fundamentals we agreed on using html attributes with `e2e-` prefix. Maybe we can apply that to the new ui tests on this pr

File: log-ui/src/app/search-form/search-form.component.ts [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+            line: '',
+          },
+        ],
+      },
+    ],
+  };
+
+  searchResult = this.emptySearchResult;
+
+  @Input() labels = { values: [] };
```
Comment (jesusreal) [inline log-ui/src/app/search-form/search-form.component.ts]: I would put the input to the top of the component

File: log-ui/src/app/search-form/search-form.component.ts [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+        switchMapTo(observables.loaded),
+        debounceTime(400),
+        distinctUntilChanged(),
+        takeUntil(observables.ngOnDestroy),
+      )
+      .subscribe(value => this.loadLabels());
+    return proxy;
+  }
+
+  ngOnInit(): void {}
```
Comment (jesusreal) [inline log-ui/src/app/search-form/search-form.component.ts]: these two lifecycle hooks can be removed

File: log-ui/src/app/search-form/search-form.component.ts (hunk 5 above)
Comment (jesusreal) [reply log-ui/src/app/search-form/search-form.component.ts]: Discussed with Peter, such refactorings will be done once integration is achieved.

File: log-ui/src/app/search-form/search-form.component.ts (hunk 6 above)
Comment (jesusreal) [reply log-ui/src/app/search-form/search-form.component.ts]: Discussed with Peter, such refactorings will be done once integration is achieved.

File: log-ui/src/app/search-form/search-form.component.html (hunk 4 above)
Comment (jesusreal) [reply log-ui/src/app/search-form/search-form.component.html]: Discussed with Peter, such refactorings will be done once integration is achieved.

**Pull request merged**